tools/retry

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 # shellcheck disable=SC2048,SC2086
 
-RETRY="${RETRY:-0}"
+RETRY="${RETRY:-3}"
```

Comment:
3 retries by default? Do you want to drop the explicit ones then?

Reply:
hm, hm, hm … no :)

Reply:
What then?

Reply:
Actually it is different. At the top of Makefile we set `RETRY=0` meaning that we still need the explicit `RETRY=3` in case where we actually need retrying. But outside of the Makefile as RETRY is not set the default is used, e.g. in the circleci scripts

Reply:
Right, so the `Makefile` reverses the default logic. Unfortunately confusing but makes sense indeed.